src/Tools/OccurenceDictionary.cs

```diff
@@ -1,21 +1,21 @@
 using System.Collections.Generic;
-using System.Linq;
```

Comment:
Oh, yeah, please! There are too many places I've used LINQ... I'll have to remove most of them when I'm done with the core alg, it'll also make the transition to C++ easier.